map/place_page_info.cpp

```diff
@@ -162,6 +162,11 @@ std::string Info::FormatSubtitle(bool withType) const
   if (GetWheelchairType() == ftraits::WheelchairAvailability::Yes)
     append(kWheelchairSymbol);
 
+  auto const fee = GetMetadata(feature::Metadata::FMD_FEE);
+  if (!fee.empty())
+    if (fee == "yes")
+      append("💰");
```

Comment:
Plese move this logic to new method `bool HasFee()`. See how `bool HasWifi()` is implemented as an example.

Reply:
Moved logic to GetLocalizedFee()